server/graphql/invoice/src/mutations/outbound_shipment/add_from_master_list.rs

```diff
@@ -0,0 +1,347 @@
+use async_graphql::*;
+use graphql_core::{
+    simple_generic_errors::{CannotEditInvoice, RecordNotFound},
+    standard_graphql_error::validate_auth,
+    standard_graphql_error::StandardGraphqlError,
+    ContextExt,
+};
+use graphql_types::{generic_errors::MasterListNotFoundForThisStore, types::InvoiceLineConnector};
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    errors::AddFromMasterListError as ServiceError,
+    invoice::outbound_shipment::AddFromMasterList as ServiceInput,
+};
+
+#[derive(InputObject)]
+pub struct AddToShipmentFromMasterListInput {
```

Comment:
ISeeWhySomeOfTheStructNamesAreReallyLongNow.. hard not to really

Reply:
Haha, we've tried to reduce the structure name with `graphql(rename` for some common things, but yeah Graphql types are quite long, I think it's ok, alternative is to abbreviate, which would be confusing